.github/workflows/appimage.yml

```diff
@@ -53,13 +53,19 @@ jobs:
       with:
         credentials_json: '${{ secrets.GCP_CREDENTIALS }}'
 
+    - name: Build Web Artifacts (PR)
+      if: contains(github.event.pull_request.labels.*.name, 'appimage') || contains(github.event.pull_request.labels.*.name, 'appimage-ignore-tests')
+      run: |
+        echo "running 'make build-web'"
```

Comment:
I added this because `make build-web` doesn't have any output for about 1min. I figured it would give everyone confidence that it was running instead of stuck.